web/src/components/organisms/Asset/AssetList/hooks.ts

```diff
@@ -90,11 +82,11 @@ export default (projectId?: string) => {
   const createAssets = useCallback(
     (files: UploadFile<File>[]) =>
       (async () => {
-        if (!projectId || !currentUser?.id) return;
+        if (!projectId) return;
         const results = await Promise.all(
           files.map(async file => {
             const result = await createAssetMutation({
-              variables: { projectId, createdById: currentUser?.id, file },
+              variables: { projectId, file },
```

Comment:
By removing the createdById, how do we know who created it? (Especially now that we have createdBy and createdByType, don't we need to send created by Id AND the type?

Reply:
There is no need to pass `createdById` as an argument from the front end when uploading a new asset any more.
createdById is being added from the operator in the backend instead.